frontend/src/features/Mission/components/MissionList/hooks/useGetFilteredMissionsQuery.ts

```diff
@@ -109,8 +107,14 @@ export const useGetFilteredMissionsQuery = (): {
       return []
     }
 
+    if (listSeaFrontGroup === ALL_SEA_FRONT_GROUP) {
```

Comment:
```suggestion
    if (listSeafrontGroup === ALL_SEA_FRONT_GROUP) {
```
Car `seafront` ce n'est qu'un seul mot (mais bon je vois quand dans le frontend c'est pas mal mis en pascalCase :/)